File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import com.pinyougou.pojo.TbBrand;

import entity.PageResult;
import entity.Result;

/**
 * 品牌接口
 * @author Administrator
 *
 */
public interface BrandService {
  public List<TbBrand> findAll();
  
  public PageResult findPage(int pageNum,int pageSize);
  
  
  public void add(TbBrand brand);
  
  public TbBrand findOne(Long id);
  
  public void update(TbBrand brand);
  
  public void delete(Long[] ids);
  
  public PageResult findPage(int pageNum,int pageSize,TbBrand brand);
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function($http){
    	this.findAll=function(){
    		return $http.get('../brand/findAll.do');
    	}
    	
    	this.findPage=function(pageNum,pageSize){
    		return $http.get('../brand/findPage.do?page='+pageNum+'&rows='+pageSize);
    	}
    	
    	this.findOne=function(id){
    		return $http.get('../brand/findOne.do?id='+id);
    	}
    	
    	this.dele=function(selectIds){
    		return $http.get('../brand/delete.do?ids='+selectIds);
    	}
    	
    	this.search=function(pageNum,pageSize,searchEntity){
    		return $http.post('../brand/search.do?page='+pageNum+'&rows='+pageSize,searchEntity);
    	}
    	
    	this.add=function(entity){
    		return $http.post('../brand/add.do',entity);
    	}
    	
    	this.update=function(entity){
    		return $http.post('../brand/update.do',entity);
    	}
    });